install/update_91_92.php

```diff
@@ -72,7 +72,11 @@ function update91to92() {
    }
 
    //put you migration script here
-
+   // Issue #1250 - Add decimal to monitor size
+   $query = "ALTER TABLE `glpi_monitors`
```

Comment:
for consistency, use Migration class to alter colum, (if possible, always avoid sql)
You can use: 
```php
$migration->changeField(...)
```